src/makeCalculator.js

```diff
@@ -4,7 +4,51 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  function operate(callback, number) {
+    result = callback(result, number);
+
+    return this;
```

Comment:
The `operate` method should use the `+=`, `-=`, `*=`, `/=` operators as appropriate in the callback functions to modify the `result` directly, instead of reassigning it.